src/components/Func/FuncParams.js

```diff
@@ -38,8 +38,12 @@ export default class FuncParams extends Component {
 
   state = initialState
 
-  componentWillMount() {
+  async componentWillMount() {
     const { func } = this.props
+    const runtimes = await RuntimeHelper.getAllRuntimes()
```

Comment:
The `componentWillMount` method is executed before the first render of the component. This will block the UI and keep it empty or in the previous state before show the new version. Instead of loading async resources in `componentWillMount` we should display a `Loading` component and call the server in `componentDidMount`.